### ISSUE TITLE
KCV loads quite slowly

### ISSUE THREAD

**Issue by Omegaxis8009** [Yuubari/KanColleViewer#11]
Recently KCV takes a long time to load up and eventually when it does load in, each new screen takes a while to load up as well, every action seems to take at least a few seconds longer

I tried it in IE and it loaded fine, but KCV takes forever, I cleared the cache and set the region cookies but they don't seem to do anything

**Comment by Yuubari** [comment]
Does Grabacr07's build also perform in the same way?

**Comment by Daneja304** [comment]
Jumping in on this since ive have the same issue as of today.
I tried running Grabacr07's build and it seemed to load just as slow but i couldnt do extensive testing because i couldnt get past the DMM region lock.

**Comment by Yuubari** [comment]
Grabacr07's original KCV should use the same region cookies though.

Then again if it's not faster than mine I cannot do anything about it. It's most likely due to nekoxy.

**Comment by Daneja304** [comment]
Hmm it seems to have fixed itself because its fast again.
Thanks for your hard work.

**Comment by SirDiazo** [comment]
As I had this same issue, I just want to comment that what resolved it for me was updating everything.

Specifically, I updated:
Windows Updates
Net 4.6.1
Flash Player (as in going to adobe.com and downloading the latest installer)
Video Card drivers (doubt this affects KCV, but I did them at the same time so I'm going to mention them.)

After that, everything is nice and snappy when I click in-game.

Note that while watching process manager, there was a "Microsoft Net Optimisation" process that ran for a few minutes after I installed Net 4.6.1, if that is still running when you try to run KCV I could see it still causing slow downs until it had finished.
